hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -1876,16 +1879,31 @@ public boolean checkLeader() {
     }
   }
 
+  public void setScmReadOnlyAdmins(OzoneAdmins scmReadOnlyAdmins) {
+    this.scmReadOnlyAdmins = scmReadOnlyAdmins;
```

Comment:
Would be nice to omit this, the test can set configuration via `OzoneConfiguration`.